source/game/kart/KartMove.hh

```diff
@@ -11,29 +11,42 @@ public:
 
     virtual void calcTurn();
     virtual void calcWheelie() {}
+    virtual void cancelWheelie() {}
```

Comment:
```suggestion
```

`cancelWheelie` does not exist in the virtual function table for karts, and thus can be made non-virtual.